app-misc/pip3line/pip3line-3.4.2.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} )
+
+inherit cmake-utils python-r1
+
+DESCRIPTION="Raw bytes manipulation, transformations (decoding and more) and interception"
+HOMEPAGE="https://github.com/metrodango/pip3line"
+
+KEYWORDS=""
+SRC_URI=""
+
+if [[ ${PV} == 9999* ]] ; then
+    inherit git-r3
+    EGIT_REPO_URI="https://github.com/metrodango/pip3line.git"
+    EGIT_BRANCH="master"
+else
+    SRC_URI="https://github.com/metrodango/pip3line/archive/v${PV}.tar.gz  -> ${P}.tar.gz"
+    KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+IUSE="distorm openssl qscintilla"
+
+RDEPEND="
+    ${PYTHON_DEPS}
+    dev-qt/qtconcurrent:5
+    dev-qt/qtcore:5
+    dev-qt/qtgui:5
+    dev-qt/qtnetwork:5
+    dev-qt/qtsvg:5
+    dev-qt/qtwidgets:5
+    dev-qt/qtxmlpatterns:5
+    distorm? ( dev-libs/distorm64 )
+    openssl? ( dev-libs/openssl:0= )
+    qscintilla? ( x11-libs/qscintilla )"
+
+DEPEND="${RDEPEND}"
+
+src_configure() {
+
+    if [[ "${PYTHON_TARGETS}" == *python3* ]]; then
```

Comment:
have you looked at `python_is_python3` from `python-utils-r1`?

Reply:
nope, what does that do ?